<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="TCOE-7374J0N-LX" name="scenarios.MyStore_SignInWithUnRegisteredUserTest" tests="1" failures="1" timestamp="16 Oct 2019 16:38:11 GMT" time="52.116" errors="0">
  <testcase name="testScenario" time="52.116" classname="scenarios.MyStore_SignInWithUnRegisteredUserTest">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	in My Store - Main Page page expected [true] but found [false],
	in My Store - Main Page page expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
in My Store - Main Page page expected [true] but found [false],
in My Store - Main Page page expected [true] but found [false]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
at auto.framework.reportlog.ReportLogInstance.assertAll(ReportLogInstance.java:122)
at auto.framework.ReportLog.assertAll(ReportLog.java:109)
at auto.framework.ReportLog$Listener.onTestSuccess(ReportLog.java:535)
at org.testng.internal.TestListenerHelper.runTestListeners(TestListenerHelper.java:70)
at org.testng.internal.Invoker.runTestListeners(Invoker.java:1388)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:633)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- testScenario -->
</testsuite> <!-- scenarios.MyStore_SignInWithUnRegisteredUserTest -->
